cmd/query/app/http_handler.go

```diff
@@ -433,10 +439,21 @@ func (aH *APIHandler) getTrace(w http.ResponseWriter, r *http.Request) {
 	if !ok {
 		return
 	}
-	// TODO: add start time & end time
 	query := spanstore.GetTraceParameters{
 		TraceID: traceID,
 	}
+	startTime := r.FormValue(startTimeParam)
+	if startTime != "" {
```

Comment:
no need to leak variable to outer scope
```suggestion
	if startTime := r.FormValue(startTimeParam); startTime != "" {
```